plugins/ag-charts-generate-code-reference-files/src/doc-interfaces/types-utils.ts

```diff
@@ -141,11 +141,13 @@ export class TypeMapper {
             } else if (h.type === 'Omit' || h.type === 'Pick' || h.type === 'Required') {
                 const n = this.resolveTypeRef(h);
                 node.members.push(...n.members);
-            } else if (h.type === 'Readonly') {
+            } else if (h.type === 'Readonly' && h.typeArguments) {
```

Comment:
nit: was it necessary? I can't think of a case in which you'll use `Readonly` without it's argument, TS won't allow it far before this script will be executed.